src/index.ts

```diff
@@ -10,43 +10,101 @@ import {
 } from "@aptos-labs/wallet-adapter-core";
 import { TxnBuilderTypes, Types } from "aptos";
 
-// CHANGE AptosWindow
+/**
+ * WARNING - This template is for legacy Wallet plugins. 
+ * This is NOT the recommended way to implement a Wallet Adapter plugin currently.
+ * 
+ * Instead, please follow the instructions here for implementing an AIP-62 compliant wallet:
+ * https://aptos.dev/en/build/sdks/wallet-adapter/wallets
+ * 
+ * AIP-62 compliant wallets are backwards compatible with this legacy template.
```

Comment:
this is a bit misleading - the wallet adapter is backwards compatible, i.e it still provides support for both aip-62 and non-aip-62. But the wallet interface is not backwards compatible and requires different implementation in some core functionalities